src/full/Agda/TypeChecking/Coverage/Cubical.hs

```diff
@@ -262,7 +262,7 @@ createMissingTrXTrXClause q_trX f n x old_sc = do
     abstractN (xTelI `applyN` g1) $ \ p -> do
     abstractT "ψ" (pure interval) $ \ psi -> do
     abstractN (xTelI `applyN` g1) $ \ q -> do
-    abstractT "x0" (pure dT `applyN` g1 `applyN` (flip map q $ \ f -> f <@> pure iz)) $ \ x0 -> do
+    abstractT "x0" (pure dT `applyN` g1 `applyN` flip map q (\ f -> f <@> pure iz)) $ \ x0 -> do
```

Comment:
From here, it makes sense to also get rid of `flip` by putting `q` after the `(\ f -> ...)`.

Or, better, use `Agda.Utils.Functor.for` for `flip map`.